src/Nethermind/Nethermind.Evm.Test/InvalidOpcodeTests.cs

```diff
@@ -131,28 +131,44 @@ public class InvalidOpcodeTests : VirtualMachineTestsBase
                                     LondonInstructions.Union(
                                     new Instruction[]
                                         {
-                                            Instruction.TLOAD,
-                                            Instruction.TSTORE,
                                             Instruction.PUSH0
                                         }
                                     )
                             )))))).ToArray();
 
+        private static readonly Instruction[] CancunInstructions =
+            FrontierInstructions.Union(
+                HomesteadInstructions.Union(
+                    ByzantiumInstructions.Union(
+                        ConstantinopleFixInstructions.Union(
+                            IstanbulInstructions.Union(
+                                BerlinInstructions.Union(
+                                    LondonInstructions.Union(
+                                        ShanghaiInstructions.Union(
+                                        new Instruction[]
+                                            {
+                                                Instruction.TSTORE,
+                                                Instruction.TLOAD
+                                            }
+                                        )
+                            ))))))).ToArray();
```

Comment:
btw if there isn't some Concat method that would take just all of those, we should add one :)

Reply:
Just realized the uselessness of all of these extra unions. you only need one union per fork because it builds on the one before it.